Source/Core/Core/Core.cpp

```diff
@@ -671,6 +671,22 @@ void SaveScreenShot()
 		SetState(CORE_RUN);
 }
 
+void SaveScreenShot(std::string name)
+{
+	const bool bPaused = (GetState() == CORE_PAUSE);
+
+	SetState(CORE_PAUSE);
+
+	const std::string& gameId = SConfig::GetInstance().GetUniqueID();
+
+	std::string filePath = File::GetUserPath(D_SCREENSHOTS_IDX) + gameId + DIR_SEP_CHR + name + ".png";
```

Comment:
Are we guaranteed to have generated a gameID folder here?
The other function's GenerateScreenshotName() first attempts to generate the full path before hand.